setup.py

```diff
@@ -22,7 +22,7 @@
     'netaddr',
     'passlib',
     'pycryptodome>=3.6.6',  # security update
-    'PyGithub>=1.43.3',
+    'PyGithub',
```

Comment:
This pin didn't really need to be removed since it was a lower bound, right?

Reply:
Yeah. But I thought it might have been added to fix a security issue years back, since its a long time, no need to keep that pin around.